### ISSUE TITLE
why  I got the same  loss and  acc

### ISSUE THREAD

**Issue by myw8** [ppplinday/tensorflow-vgg16-train-and-test#1]
I  just change the pictures  but  got the loss  and  acc  are same..
![eebbdfff-75ad-45f6-bdac-eb881f4cf056](https://user-images.githubusercontent.com/17568519/28571983-c7824f76-7177-11e7-88d5-69328d3d4582.png)

**Comment by Lvjiming** [comment]
yes，I meet the same problem

**Comment by ppplinday** [comment]
I think there are some problem in my program and I will fix it later. Sorry about that.

**Comment by star5o** [comment]
> I think there are some problem in my program and I will fix it later. Sorry about that.

Sorry, I meet the problem now. How can I fix it？
